linode/manager
# feat: [M3-6941]- Uniform Spacing b/w Resource Link Columns In Empty State Landing Pages

## Description 📝
This PR addresses the issue with a lack of uniform spacing between the **Getting Started Guides** and **Video Playlist** sections on the empty state landing pages.

## Changes  🔄
List any change relevant to the reviewer.
- Updated the `width` of the `ResourcesLinkSection` by adding a breakpoint declaration for `mid`.
- Replaced default export for `KubernetesLanding` with named export.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/user-attachments/assets/7c37e1bf-b986-4ba2-8181-7ffc138356ed" /> | <video src="https://github.com/user-attachments/assets/b274d863-6f9a-4dcd-9cca-0c152f23cd26" /> |

## How to test 🧪
### Prerequisites
(How to setup test environment)
- Have a user account without any compute instances in order to display the empty state landing pages for each entity.

### Verification steps
(How to verify changes)
- Compare the spacing between the **Getting Started Guides** and **Video Playlist** sections on the empty state landing pages in the develop branch against this PR.
- Verify that the spacing is now uniform across entities with both **Getting Started Guides** and **Video Playlist** sections.
  - Refer to the videos in the **Preview** section to verify changes.

## As an Author I have considered 🤔
